### ISSUE TITLE
Is Paper Available?

### ISSUE THREAD

**Issue by huixiancheng** [mapping520/SAMe3d#1]
As shown in the question.

**Comment by mapping520** [comment]
Yes, the paper has been accepted. But there are still issues with the code, and we are modifying it. Thank you for your attention.
